test/ejabberd_tests/tests/amp_SUITE.erl

```diff
@@ -577,14 +585,16 @@ drop_deliver_to_offline_user_test(Config) ->
               client_sends_message(Alice, Msg),
 
               % then
-              case lists:member(Rule, Rules) orelse ?config(offline_storage, Config) /= offline_failure of
+              case lists:member(Rule, Rules) orelse
+                         ?config(offline_storage, Config) /= offline_failure of
                   true -> client_receives_nothing(Alice);
                   false -> client_receives_generic_error(Alice, <<"500">>, <<"wait">>)
               end,
 
               % then
               case is_offline_storage_working(Config) andalso not lists:member(Rule, Rules) of
-                  true -> user_has_incoming_offline_message(FreshConfig, bob, <<"A message in a bottle...">>);
+                  true -> user_has_incoming_offline_message(FreshConfig, bob,
+                                                <<"A message in a bottle...">>);
```

Comment:
I see the binary is used few lines above. Let's assign it to a var and use var instead.